services/app-web/src/common-code/healthPlanFormDataType/UnlockedHealthPlanFormDataType.d.ts

```diff
@@ -66,6 +66,208 @@ type RateCapitationType = 'RATE_CELL' | 'RATE_RANGE'
 
 type ManagedCareEntity = 'MCO' | 'PIHP' | 'PAHP' | 'PCCM'
 
+type Day =
+    | '01'
+    | '02'
+    | '03'
+    | '04'
+    | '05'
+    | '06'
+    | '07'
+    | '08'
+    | '09'
+    | '10'
+    | '11'
+    | '12'
+    | '13'
+    | '14'
+    | '15'
+    | '16'
+    | '17'
+    | '18'
+    | '19'
+    | '20'
+    | '21'
+    | '22'
+    | '23'
+    | '24'
+    | '25'
+    | '26'
+    | '27'
+    | '28'
+    | '29'
+    | '30'
+    | '31'
+
+type Month =
+    | '01'
+    | '02'
+    | '03'
+    | '04'
+    | '05'
+    | '06'
+    | '07'
+    | '08'
+    | '09'
+    | '10'
+    | '11'
+    | '12'
+
+type Year =
```

Comment:
Might be worth a comment here about how this is a potentail place for bugs if dates outside of these years get entered.

Reply:
resolved offline,  no change